src/Entity/BookingStatus.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\BookingStatusRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass=BookingStatusRepository::class)
+ */
+class BookingStatus
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private int $id;
+
+    /**
+     * @ORM\Column(type="string", length=255)
```

Comment:
vu que tu as fais un form à la main coté vu, tu dois refaire une validation supplémentaire ici
https://symfony.com/doc/current/reference/constraints/Choice.html#supplying-the-choices-with-a-callback-function
(quand tu utilises le component form, c'est fait en automatiqeu)